### DISCUSSION TITLE
Add a popout player

### DISCUSSION THREAD

**Discussion by KristofferTroncoso** [KristofferTroncoso/ThemeSong#8] [Ideas and feature requests]
Add a popout player like in Spotify

**Comment by hanubeki** [comment]
I think "Document Picture-in-Picture" can make popout player, I'm not sure how it's difficult to add though.
https://developer.chrome.com/docs/web-platform/document-picture-in-picture/

**Reply by KristofferTroncoso** [reply]
Apparently its easy for when it's playing a music video. But i'll have to make a canvas workaround for non-video songs. Thanks!

**Comment by hanubeki** [comment]
SynQ, another extension for YouTube Music (and other streaming services) has popout player that actually works.
https://chromewebstore.google.com/detail/synq-for-spotify-youtube/jcdmcpefbebkldopabclefjaggejocff
